addons/scratchr2/profile.css

```diff
@@ -98,25 +99,25 @@
 .activity-stream .actor a {
   color: #575e75 !important;
 }
-#content > .box:not(#profile-data):not(.slider-carousel-container):not(#page-404) {
+#content > .box:not(#profile-data):not(.slider-carousel-container):not(.sa-stats):not(#page-404) {
   border-radius: 0;
   border: none;
   background: rgba(77, 151, 255, 0.1);
   padding-top: 16px;
   padding-bottom: 16px;
   margin-bottom: 0;
 }
-#content > .box:not(#profile-data):not(.slider-carousel-container) .box-head,
-#content > .box:not(#profile-data):not(.slider-carousel-container) .box-content {
+#content > .box:not(#profile-data):not(.slider-carousel-container):not(.sa-stats) .box-head,
+#content > .box:not(#profile-data):not(.slider-carousel-container):not(.sa-stats) .box-content {
```

Comment:
I think we are trying to avoid making our own classes (if possible) and use scratch's class to avoid having different selectors and also to work for dark mode.

Reply:
There is no Scratch's class that would make it possible to exclude the box from this selector without changing it (`slider-carousel-container` changes padding).

Reply:
Yeah, I understand, just checking.